frontend/src/components/my-grant-profile-page-handler/sidebar.tsx

```diff
@@ -34,9 +34,17 @@ export const Sidebar = ({
       </GrantInfo>
 
       <GrantInfo label={<FormattedMessage id="profile.myGrant.grantType" />}>
-        <FormattedMessage
-          id={`grants.form.fields.grantType.values.${grantType}`}
-        />
+        <VerticalStack gap="small">
+          {grantType.map((type, index) => (
+            <Text size="label2" weight="strong">
+              {" "}
```

Comment:
why the empty space? maybe a vertical Spacer works better?

Reply:
no was a mistake, think my editor put it somehow